### DISCUSSION TITLE
Feature request NCICT

### DISCUSSION THREAD

**Discussion by choonsiklee** [ncidose/ncidosetools#31] [NCICT] [answered]
Please leave new feature requests here.

**Comment by choonsiklee** [comment]
Add more scanner models to the scanner list which requires the nCTDIw (head and body CTDI phantoms) values for 80, 100, 120, and 140 kV and head/body filters for 1 cm collimator.

**Reply by fgardavaud** [reply]
Hello,
Add spectral acquisition mode (e.g fast-kV switching)

**Comment by andreasjahnen** [comment]
NCICT Batch mode does need a CTDIvol to operate. It happens quite often, that the CTDIvol is not known for an examination. It would be cool to have a batch mode, that can estimate the CDTIvol from the make, model, and technical parameters, as available in the GUI. 

This function was available in NCICT 1.0 beta if I can remember correctly. 

Best Regards,
Andreas

**Comment by amalmaimani** [comment]
Could you add Siemens SOMATOM force 256 slice and GE revolution CT 256 please.

**Comment by choonsiklee** [comment, marked as answer]
Add an option to select the ICRP reference phantoms (including the newborn) from batch mode

**Comment by choonsiklee** [comment]
display the height and weight of the fetus and pregnant phantoms

**Comment by choonsiklee** [comment]
organ dose calculation for the x-ray spectrum with Tin filters

**Reply by mkb600** [reply]
Spectral filtering in general, i.e. Tin filters, Silver filter (silver beam Canon).

**Comment by choonsiklee** [comment]
a feature to turn on and off organ-based tube current modulation

**Comment by sam-alam** [comment]
Please add GE Revolution/-EVO/-Apex series

**Comment by pvdtpvdt** [comment]
If possible, please add  the newer CT ONE model of Canon (genesis/prism/insight) and the Canon Precision system
Also, more paramteers in batch mode (like CT model and pitch) would be helpfull. Thank you!

**Comment by diogomadaleno** [comment]
Is it possible to add some information about new scanners?
Siemens Go Top/go all?
GE Revolution, Apex?

I understand that it is difficult to add these features since the market offers increasingly faster scanners.

**Reply by choonsiklee** [reply]
We apologize for the delayed response. We have recently redesigned the CTDI library structure to facilitate the incorporation of CTDI values from newly released scanners, and we are planning a CTDI survey to collect user-contributed data after the upcoming official release of NCICT4 with detailed cardiac models. Please stay tuned.

**Comment by ncidoseteam** [comment]
Add support for locale-specific decimal separators in Windows (e.g., comma vs. dot) to ensure consistent numeric input and correct TCM handling.

**Comment by ncidoseteam** [comment]
Add an option to select the phantom based on the patient’s water-equivalent diameter (WED)

**Comment by ncidoseteam** [comment]
Bug Report: Displayed DLP does not match CTDIvol × Scan Length

**Reply by ncidoseteam** [reply]
Fixed
